src/Service.Host/client/src/components/StockPools.js

```diff
@@ -0,0 +1,309 @@
+                    <DialogContent>
+                        <Search
+                            autoFocus
+                            propertyName={`${c.field}-searchTerm`}
+                            label="defaultLocation"
+                            resultsInModal
+                            resultLimit={100}
+                            value={searchTerm}
+                            loading={storageLocationsLoading}
+                            handleValueChange={(_, newVal) => setSearchTerm(newVal)}
+                            search={() => getStorageLocations(null, `?searchTerm=${searchTerm}`)}
+                            searchResults={storageLocationsResult?.map(s => ({
+                                ...s,
+                                id: s.name
+                            }))}
+                            priorityFunction="closestMatchesFirst"
+                            onResultSelect={handleSearchResultSelect}
+                            clearSearch={() => {}}
+                        />
+                    </DialogContent>
+                    <DialogActions>
+                        <Button onClick={handleClose}>Close</Button>
+                    </DialogActions>
+                </Dialog>
+            </div>
+        );
+    };
+
+    const stockPoolColumns = [
+        {
+            field: 'stockPoolCode',
+            headerName: 'Code',
+            editable: params => params.row?.creating === true,
+            width: 100
+        },
+        { field: 'stockPoolDescription', editable: true, headerName: 'Description', width: 225 },
+        {
+            field: 'dateInvalid',
+            headerName: 'Date Invalid',
+            editable: true,
+            width: 175,
+            valueFormatter: value => value && moment(value).format('DD-MMM-YYYY')
+        },
+        { field: 'stockCategory', editable: true, headerName: 'Stock Category', width: 100 },
+        {
+            field: 'accountingCompanyCode',
+            editable: true,
+            headerName: 'Accounting Company',
+            width: 100,
+            defaultFormat: true,
+            valueOptions: accountingCompany?.map(ac => ac.name),
+            type: 'singleSelect'
+        },
+        {
+            field: 'defaultLocation',
+            headerName: 'Default Location',
+            width: 150,
+            editable: true,
+            type: 'search',
+            search: getStorageLocations,
+            searchResults: storageLocationsResult,
+            searchLoading: storageLocationsLoading,
+            searchUpdateFieldNames: [
+                { fieldName: 'defaultLocation', searchResultFieldName: 'locationId' },
+                { fieldName: 'defaultLocationName', searchResultFieldName: 'locationCode' }
+            ],
+            clearSearch: clearStorageLocations,
+            renderCell: searchRenderCell
+        },
+        { field: 'sequence', editable: true, headerName: 'Sequence', width: 100 }
```

Comment:
not displaying the include in mrp field which is really the only reason they'll need this form